html/img.html

```diff
@@ -5,5 +5,62 @@
 <img src="pic_trulli.jpg" alt="Trulli" width="700" height="333">
 
 
+</body>
+</html>
+
+<html>
+<body>
+
+<h2>Image Size</h2>
+
+<p>Here we use the style attribute to specify the width and height of an image:</p>
+
+<img src="img_girl.jpg" alt="Girl in a jacket" style="width:500px;height:600px;">
+
+<html>
+<body>
+
+<h2>Image Size</h2>
+
+<p>Here we specify the width and height of an image with the width and height attributes:</p>
+
+<img src="img_girl.jpg" alt="Girl in a jacket" width="500" height="600">
+
+</body>
+</html><!DOCTYPE html>
+<html>
```

Comment:
Problem:
- This html file has multiple **html** **body** and **head** tags, one html file should have just one html one body and one head

Reply:
I will resolve this problem

Reply:
👍🏻

Reply:
👍🏻